frontend/encore/faq.rst

```diff
@@ -0,0 +1,80 @@
+FAQ and Common Issues
+=====================
+
+My App Lives under a Subdirectory
+---------------------------------
+
+If your app does not live at the root of your web server (i.e. it lives under a subdirectory,
+like ``/myAppSubdir``), you just need to configure that when calling ``Encore.setPublicPrefix()``:
+
+.. code-block:: diff
+
+    // webpack.config.js
+    Encore
+        // ...
+
+        .setOutputPath('web/build/')
+
+        - .setPublicPath('/build')
+        + // this is your *true* public path
+        + .setPublicPath('/myAppSubdir/build')
+
+        + // this is now needed so that your manifest.json keys are still `build/foo.js`
+        + // i.e. you won't need to change anything in your Symfony app
+        + config.setManifestKeyPrefix('build')
```

Comment:
`config` should not be here, as you add this in a chain call (and there is no `config` variable)